webapp/src/modules/wallet/utils.js

```diff
@@ -23,11 +25,19 @@ export function getWalletSagaOptions() {
       ),
       new Marketplace(env.get('REACT_APP_MARKETPLACE_CONTRACT_ADDRESS')),
       new EstateRegistry(env.get('REACT_APP_ESTATE_REGISTRY_CONTRACT_ADDRESS')),
+      new ERC721Bid(env.get('REACT_APP_ERC721_BID_CONTRACT_ADDRESS')),
```

Comment:
Why is this here if why are adding it conditionally below?

Also, do you think it might look nicer if we extract this like we did with Mortgage and Auction?

I know it's only one contract but we can keep a single `return` (instead of `const options`) and once bid goes live it's just a matter of replacing the method with this line here.

And, the method is really simple
```
function getBidContracts() {
  return isFFEnabled() ? [contract] : []
}